File: go/gin/internal/modules/client/dto/find_by_cpf_client_dto.go
package dto

import "tcc/internal/common/helper"

type FindByCPFClientDto struct {
	CPF string `json:"cpf" validate:"required,cpf"`
}


func (dto *FindByCPFClientDto) Transform() {
	// CPF
	dto.CPF = helper.CleanDigits(dto.CPF)
}

File: rust/axum/src/modules/client/dto/find_by_cpf_client_dto.rs
use serde::{Deserialize, Serialize};
// use validator::Validate;
// use crate::internal::common::helper;

#[derive(Debug, Deserialize, Serialize)]
pub struct FindByCPFClientDto {
    // #[validate(custom = "helper::validate_cpf")]
    pub cpf: String,
}

impl FindByCPFClientDto {
    pub fn transform(&mut self) {
        // CPF
        // self.cpf = helper::clean_digits(&self.cpf);
    }
}
